toonz/sources/stdfx/iwa_fractalnoisefx.h

```diff
@@ -50,60 +50,60 @@ class Iwa_FractalNoiseFx final : public TStandardZeraryFx {
   };
 
 protected:
-  // Fractal Type �t���N�^���̎��
+  // Fractal Type �t���N�^���̎��
```

Comment:
The typo fixes (related to translations) do not appear to be accurate in this header file.

Reply:
Other than the suggested changes to iwa_fractalnoisefx.h that aren't correct all other typo fixes appear to be correct and trivial in nature.  That is to say they are text and/or comments which can be merged without problem.  It would be good to either drop that specific file's changes or alternatively, translate those comments into English.

Reply:
Sorry about that! It's some damn setting in vscode that is doing that. Do any native speakers want to translate those comments and push the changes to this PR?

Reply:
They already have translations (the English to the left). You can just undo these changes.

Reply:
yikes, my git editor (gitkraken) **doesn't register** that these changes even exist :fearful:

Reply:
@konero I've reverted these specific changes in d5a2697